test/dotnet-publish.Tests/PublishTests.cs

```diff
@@ -92,6 +92,29 @@ public void FailWhenNoRestoreTest()
             publishCommand.Execute().Should().Fail();
         }
 
+        [Theory]
+        [InlineData("ubuntu.14.04-x64", "", "libhostfxr.so", "libcoreclr.so", "libhostpolicy.so")]
```

Comment:
My reasoning was that covering each of the expected asset types `.so, .dll, .dylib`, would cover most situations, but I can see that leaves a testing gap related to packaging issues for a specific platform.

Will add it in, thanks